### PR TITLE
Remove jQuery dependency from commands layout

### PR DESCRIPTION
**Pull request by tnir**
### What was the end-user problem that led to this PR?

An end-user has loaded unused jQuery.

- See also #653

### What was your diagnosis of the problem?

These code currently depending on jQuery can be replaced with vanilla JS.

### What is your fix for the problem, implemented in this PR?

The relevant code was simply replaced with vanilla JS.

### Why did you choose this fix out of the possible options?

Just follow the instruction from AnchorJS: https://github.com/bryanbraun/anchorjs/blob/master/README.md

HTMLs for command references were done from ones for search with Lunr.

Signed-off-by: Takuya Noguchi <takninnovationresearch@gmail.com>

### CODE REVIEW

**Comment by olleolleolle** [comment]
Looks grand. Nice removal.

File: assets/javascripts/commands_layout.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
+el.addEventListener('input', function (e) {
+  document.location.href = e.target.value;
 });
 
-$(document).ready(function() {
-  anchors.add('#page-content-wrapper h1, #page-content-wrapper h2, #page-content-wrapper h3, ' +
-    '#page-content-wrapper h4, #page-content-wrapper h5');
-});
+anchors.add('#page-content-wrapper h1, #page-content-wrapper h2, #page-content-wrapper h3, ' +
+  '#page-content-wrapper h4, #page-content-wrapper h5');
```
Comment (olleolleolle) [inline assets/javascripts/commands_layout.js:11]: Minor: Perhaps we can use some formatting spaces to show that these strings are _very_ alike? E.g. one selector on each line, or something. Still as a selector, but formatted for easy scanning and understanding.

```suggestion
anchors.add(
  '#page-content-wrapper h1, ' +
  '#page-content-wrapper h2, ' + 
  '#page-content-wrapper h3, ' +
  '#page-content-wrapper h4, ' + 
  '#page-content-wrapper h5'
);
```

**Review by deivid-rodriguez — APPROVED** [review]
This looks great to me.

If @tnir wants to address @olleolleolle's comment seems fine. If postponing @olleolleolle's suggestion to a different PR to minimize changes is preferred, that's also fine with me!

File: assets/javascripts/commands_layout.js (hunk 1 above)
Comment (olleolleolle) [reply assets/javascripts/commands_layout.js:11]: This is unrelated to the PR, but I just thought of this change while reading. Not important that it goes in with this PR.

**Comment by tnir** [comment]
@deivid-rodriguez Thanks. I defer it to another PR.

File: assets/javascripts/commands_layout.js (hunk 1 above)
Comment (tnir) [reply assets/javascripts/commands_layout.js:11]: Thanks again @olleolleolle 😍. Good point. I guess there is no `h5` (or `h1`) in `#page-content-wrapper`, so we should remove these.

```js
anchors.add('#page-content-wrapper h2, #page-content-wrapper h3, #page-content-wrapper h4');
```
or
```js
anchors.add(
  '#page-content-wrapper h2, ' + 
  '#page-content-wrapper h3, ' +
  '#page-content-wrapper h4'
);
```
would be the final result I am expecting. In either case, I would like to defer any change on this as possible as we can.

Comment (tnir) [reply assets/javascripts/commands_layout.js:11]: Before this heuristic improvement, I want to apply the non-heuristic way in #737, @olleolleolle. It looks like more time need for me to change this snippet.

**Pull request merged**